### PR TITLE
 fix for issue 1641 dont call select on empty layer set

### PR DESCRIPTION
**Pull request by pineapplespatula**
fix for issue #1641

### CODE REVIEW

File: src/js/actions/tools.js [hunk 1]
```diff
@@ -59,8 +59,13 @@ define(function (require, exports) {
             document.layers.all.first().isBackground) {
             return descriptor.PlayObject(defaultObj);
         }
-        var layerSpec = document.layers.allSelected.toList(),
-            layerRef = layerSpec
+        var layerSpec = document.layers.allSelected.toList();
+
+        if (layerSpec.isEmpty()) {
+            return descriptor.PlayObject(defaultObj);
```
Comment (iwehrman) [inline src/js/actions/tools.js]: Shouldn't that be `descriptor.playObject`? Above too?

Comment (iwehrman) [inline src/js/actions/tools.js]: Also, why do we pass in layer references here if it suffices to just pass in the document reference?

Comment (pineapplespatula) [inline src/js/actions/tools.js]: where are we passing layer references?

**Comment by iwehrman** [comment]
LGTM, thanks!

**Pull request merged**